radio/src/pulses/pulses_arm.cpp

```diff
@@ -65,7 +65,7 @@ void setupPulses(uint8_t port)
           required_protocol = PROTO_PXX;
           break;
 #if defined(MULTIMODULE)
-      case MODULE_TYPE_MULTIMODULE:
+        case MODULE_TYPE_MULTIMODULE:
```

Comment:
It is now the same lavel as case MODULE_TYPE_XJT and case MODULE_TYPE_PPM. Or do I miss something like tab vs spaces?